CHANGELOG.md

```diff
@@ -41,7 +42,6 @@ The following is a curated list of changes in the Enact agate module, newest cha
 - `agate/Input` prop `invalidMessage` to customize the tooltip message when `invalid` prop is true
 - `agate/Input` prop `size` to change the size of both input and icons (default large)
 - `agate/Item` prop `centered`, `disabled`, `inline`, `marqueeOn`, and `size`
-- `agate/Keypad` prop `spotlightDisabled` to disable 5-way navigation
```

Comment:
Why did you remove changelog of previous version ?

Reply:
It was by mistake.